package.json

```diff
@@ -6,13 +6,14 @@
   "scripts": {
     "start": "node index.js",
     "lint": "./node_modules/.bin/spacey-standard",
-    "test": "npm run pg:init && npm run pg:load-test-data && LABS_AUTH_SERVICE_logger_pino_level=silent ./node_modules/.bin/lab",
+    "test": "npm run app:init-test-db && LABS_AUTH_SERVICE_logger_pino_level=silent ./node_modules/.bin/lab",
     "coverage": "LABS_AUTH_SERVICE_logger_pino_level=silent ./node_modules/.bin/lab -r html -o coverage.html",
     "pg:build": "docker-compose build",
     "pg:start": "docker-compose up -d",
     "pg:stop": "docker-compose down",
-    "pg:init": "[ ! -d ./logs ] && mkdir ./logs;./scripts/init/database/db_init.sh | tee ./logs/init-db.log",
-    "pg:load-test-data": "./scripts/init/database/db_load.sh | tee ./logs/load-test-data.log"
+    "pg:init": "LABS_AUTH_SERVICE_env=test node database/init.js && node database/migrate.js --version=1",
```

Comment:
As said above, I'll change the name to somthing more meaningfull like `LABS_AUTH_SERVICE_local`. We do not actually have different envs (neither confs by envs)